manubot/manubot.py

```diff
@@ -182,6 +182,36 @@ def read_jsons(paths):
     return user_variables
 
 
+def add_author_affiliations(variables):
+    """
+    Edit variables to contain numbered author affiliations. Specifically,
+    add a list of affiliation_numbers for each author and add a list of
+    affiliations to the top-level of variables. If no authors have any
+    affiliations, variables is left unmodified.
+    """
+    rows = list()
+    for author in variables['authors']:
+        name = author['name']
+        affiliations = author.get('affiliations', [])
+        affiliations = (affiliations if isinstance(affiliations, list) else
+                        affiliations.split('; '))
```

Comment:
I believe now that an authors affiliations should always be a list in `metadata.yaml`. However, to keep backwards compatibility, `;  `-separated affiliations are accepted.

@agitter do you think affiliations should be a list in `metadata.yaml`? If so, I'll update the manubot-rootstock template after this is merged? Do we want to retain backwards compatibility and if so is this the best way?